kivy/logger.py

```diff
@@ -464,39 +575,42 @@ def is_color_terminal():
     else None
 )
 
+assert not hasattr(sys, '_kivy_logging_handler'), \
+    "Not supported. Try logging.root.addHandler()"
+
```

Comment:
Maybe we can add a reference to issue https://github.com/kivy/kivy/issues/7981, so if it happens that someone looks at this chunk of code in 1/2 releases can take the occasion to just delete this line?